operator/actions/initialize.go

```diff
@@ -89,18 +89,21 @@ func podExec(konfig *rest.Config, klient *kubernetes.Clientset, namespace string
 	req.VersionedParams(&v1.PodExecOptions{
 		Container: pod.Spec.Containers[0].Name,
 		Command:   command,
+		Stdin:     true,
 		Stdout:    true,
 		Stderr:    true,
-	}, scheme.ParameterCodec)
+	}, metav1.ParameterCodec)
 
 	exec, err := remotecommand.NewSPDYExecutor(konfig, "POST", req.URL())
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to create executor: %w", err)
 	}
 
+	stdin := &bytes.Buffer{}
 	stdout := &bytes.Buffer{}
 	stderr := &bytes.Buffer{}
-	err = exec.Stream(remotecommand.StreamOptions{
+	err = exec.StreamWithContext(ctx, remotecommand.StreamOptions{
+		Stdin:  stdin,
```

Comment:
We still error in line 113 with `you must specify at least 1 of stdin, stdout, stderr`:
```
failed to get node statuses for recommission: failed to execute: unable to upgrade connection: you must specify at least 1 of stdin, stdout, stderr
```

I just cannot figure out why that happens now, even when having added the stdin...